include/libtorrent/error_code.hpp

```diff
@@ -92,6 +92,8 @@ namespace libtorrent {
 			invalid_escaped_string,
 			// Operation is not permitted since the session is shutting down
 			session_is_closing,
+			// The session_handle is not referring to a valid session_impl
+			invalid_session_handle,
```

Comment:
the reason for keeping them stable is it may be a way to translate error messages in user code.